tests/test_minim.py

```diff
@@ -152,3 +151,42 @@ def test_relax_fixed_vol(cu_slab):
     assert atoms_opt.positions == approx(expected_positions, abs=3e-3)
 
     assert atoms_opt.info['config_type'] == 'cu_slab_minim_last_converged'
+
+
+def test_resampling_trajectory(cu_slab):
+
+    calc = EMT()
+
+    cu_slab_optimised = cu_slab.copy()
+    cu_slab_optimised.set_positions(expected_relaxed_positions_constant_pressure)
+
+    inputs = ConfigSet_in(input_configs = [cu_slab, cu_slab_optimised] )
+
+    # returns full optimisation trajectories
+    atoms_opt = minim.run_op(inputs, calc, fmax=1e-2, precon=None,
+                          logfile='-', verbose=True, pressure=-1.1215, 
+                          steps=2, traj_subselect=None)
```

Comment:
What do you think about changing the name of this variable OR the function that's called, so they either both use "subselect" or "resample"?

Reply:
I like consistency, thanks for pointing this out. I went with "resample", I think that is _slightly_ more intuitive.